readme.md

```diff
@@ -8,7 +8,7 @@
 [![Downloads](https://img.shields.io/npm/dt/zustand.svg?style=flat&colorA=000000&colorB=000000)](https://www.npmjs.com/package/zustand)
 [![Discord Shield](https://img.shields.io/discord/740090768164651008?style=flat&colorA=000000&colorB=000000&label=discord&logo=discord&logoColor=ffffff)](https://discord.gg/poimandres)
 
-A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks, isn't boilerplatey or opinionated.
+A small, fast and scalable bearbones state-management solution using simplified flux principles. Has a comfy API based on hooks, isn't Boilerplate or opinionated.
```

Comment:
I think it's an intentional made-up word, no?

Reply:
That's right, the feel of this paragraph is relaxed with some made-up words: bearbones instead of barebones and boilerplatey instead of requiring a lot of boilerplate code.

I suggest closing this PR without changing, especially that the noun "boilerplate" doesn't fit here; an adjective is needed.